File: Bjut_20160325/Scala/src/Work/Student.scala
package Work

import java.io.Serializable

/**
  * Created by hadoop on 3/22/16.
  */
class Student extends Serializable{
  var num:String = null
  var name:String = null
  var balance:Int = 0
  def this(nu:String, na:String, ba:Int){
    this()
    this.num = nu
    this.name = na
    this.balance = ba
  }
  override def toString(): String ={
    "Name: " + name + "\tNumber: " + num +"\tBalance: " + balance
  }
}

File: hadoop/src/JavaTest/Student.java
package JavaTest;

/**
 * Created by hadoop on 3/24/16.
 */
public class Student {
    private int age;
    public String name;
    public Student(){
        super();
    }
    public Student(int a, String n){
        super();
        this.age = a;
        this.name = n;
    }
    public void method(){
        System.out.printf("student method");
    }

    public void show(String name, int num) {
        System.out.printf("student name: " + name + " num:" + num);
    }

    public String toString(){
        return "age: " + age + "\nname: " + name;
    }
}
